.github/workflows/cleanup-azure-vms.yml

```diff
@@ -49,6 +49,11 @@ jobs:
     runs-on: windows-latest
     if: ${{ always() }}
     steps:
+    - name: Harden Runner
```

Comment:
I'd prefer not to add all these. We did in the past for MsQuic and never use it.

Reply:
Let me see if I can generate a more stripped-down version of this PR using the tool.